src/main.c

```diff
@@ -25,5 +26,9 @@ int main(int argc, char *argv[]) {
     string[fsize] = '\0';
 
     cJSON *json = cJSON_Parse(string);
+    if (json == NULL) {
+        exit(1);
+    }
+    build_simulation_scenario("sir", NULL, NULL, "output.json");
     return json == NULL;
 }
```

Comment:
Super small but I believe C standards have .c files ending with a newline. Shouldn't cause any problems for us but some very strict compilers might reject it.